lib_test/jbuild

```diff
@@ -1,6 +1,6 @@
 (executables
  ((names (parse_test))
-  (libraries (mirage-block-unix mirage-types-lwt oUnit lwt io-page.unix
+  (libraries (mirage-block-unix mirage-types-lwt oUnit lwt
```

Comment:
I guess travis wants to tell you that `io-page-unix` is still needed somewhere in here...